kubetest2/internal/deployers/eksapi/nodegroup.go

```diff
@@ -267,7 +284,7 @@ func (m *NodegroupManager) createUnmanagedNodegroupWithEFA(infra *Infrastructure
 			},
 			{
 				ParameterKey:   aws.String("SubnetIds"),
-				ParameterValue: aws.String(infra.subnetsPrivate[0]), // this is load bearing! EFA requires a private subnet
+				ParameterValue: aws.String(strings.Join(infra.subnetsPrivate, ",")), // this is load bearing! EFA requires a private subnet
```

Comment:
Do we have to change anything else for EFA cross-subnet to work?

https://aws.amazon.com/about-aws/whats-new/2024/07/elastic-fabric-adapter-cross-subnet-communication/

Reply:
The first subnet in the list may be in a different az than the reserved capacity. We need to have a way to choose the az that matches the reserved capacity.